### ISSUE TITLE
Cosmo DB Mongo Collection Order of compound index matters or not

### ISSUE THREAD

**Issue by yupwei68** [Azure/azure-rest-api-specs#10909]
request:
{
  "parameters": {
    "accountName": "ddb1",
    "resourceGroupName": "rg1",
    "api-version": "2020-04-01",
    "subscriptionId": "subid",
    "databaseName": "databaseName",
    "collectionName": "collectionName",
    "createUpdateMongoDBCollectionParameters": {
      "location": "West US",
      "tags": {},
      "properties": {
        "resource": {
          "id": "collectionName",
          "indexes": [
            {
              "key": {
                **"keys": ["testKey" ]** <<- Here whether the order of the keys matters
              },
              "options": {
                "expireAfterSeconds": 100,
                "unique": true
              }
            }
          ],
          ...
        },
        "options": {}
      }
    }
  },

**Comment by yupwei68** [comment]
@MehaKaushik Could you please support us on this question?

**Comment by MehaKaushik** [comment]
@yupwei68 MongoDB API for CosmosDB follows the same rules as for the MongoDB.

Referencing this https://docs.mongodb.com/manual/core/index-compound/.  
"The order of the fields listed in a compound index is important. The index will contain references to documents sorted first by the values of the item field and, within each value of the item field, sorted by values of the stock field. See Sort Order for more information."

To summarize: the order will matter.

**Comment by yupwei68** [comment]
Thanks @MehaKaushik !
